jrcc-document-access-libs/src/main/java/ca/bc/gov/open/jrccaccess/libs/TransactionInfo.java

```diff
@@ -42,7 +44,7 @@ public TransactionInfo(
 		this.sender = sender;
 		this.fileName = fileName;
 		this.receivedOn = receivedOn;
-
+		this.uuid = UUID.randomUUID();
 	}
```

Comment:
Just for style i would set the id first, i kmow it's exactly the same result but I tend to set the fields generated first the parameterized ones.

Reply:
will update